rdmo/projects/viewsets.py

```diff
@@ -163,9 +162,13 @@ def options(self, request, pk=None):
                 raise NotFound() from e
 
             # check if the optionset belongs to this catalog and if it has a provider
+            project.catalog.prefetch_elements()
```

Comment:
It could use more test covarage (https://coveralls.io/builds/64562887/source?filename=rdmo%2Fprojects%2Fviewsets.py#L156). However, there is a double try and except and otherwise returns a 404